code/modules/unit_tests/language_transfer.dm

```diff
@@ -42,8 +42,8 @@
 
 	dummy.set_species(/datum/species/lizard/silverscale)
 
-	var/list/common_sources = holder.understood_languages[/datum/language/common]
-	var/sources = ""
+	var/list/common_sources = holder.spoken_languages[/datum/language/common]
+	var/sources = "[holder.type] - "
```

Comment:
what da hell does this do?

Reply:
I'm testing stuff since my pc is a tad too slow ig